modules/vaos/spec/services/eps/base_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Eps::BaseService do
+  let(:user) { double('User', account_uuid: '1234') }
+  let(:service) { described_class.new(user) }
+
+  describe '#headers' do
+    it 'returns the correct headers' do
+      allow(RequestStore.store).to receive(:[]).with('request_id').and_return('request-id')
+
+      expected_headers = {
+        'Authorization' => 'Bearer 1234',
+        'Content-Type' => 'application/json',
+        'X-Request-ID' => 'request-id'
+      }
+      expect(service.send(:headers)).to eq(expected_headers)
```

Comment:
`headers` is not private, suggest calling it directly, rather than `.send()`

Reply:
Gotcha, that makes sense